beangulp/importers/csv.py

```diff
@@ -283,7 +292,7 @@ def get(row, ftype):
                 if txn_time is not None:
                     meta['time'] = str(dateutil.parser.parse(txn_time).time())
                 if balance is not None:
-                    meta['balance'] = self.parse_amount(balance)
+                    meta['balance'] = (self.parse_amount(balance), currency)
```

Comment:
This would look more idiomatic if written
```python
meta['balance'] = Amount(self.parse_amount(balance), currency)
```
and the rest of the code adjusted to this change.

Reply:
I missed this in the last push. I will add the changes now.